activiti-core/activiti-process-validation/src/main/java/org/activiti/validation/validator/impl/EventValidator.java

```diff
@@ -107,4 +110,15 @@ protected void handleCompensationEventDefinition(BpmnModel bpmnModel, Process pr
     }
   }
 
+  private void handleLinkEventDefinition(Process process, Event event, LinkEventDefinition linkEventDefinition, List<ValidationError> errors) {
+      FlowNode flowNode = ((FlowNode) process.getFlowElement(event.getId()));
```

Comment:
No need to retrieve the flow node again. You use directly `event.isLinkThrowEvent()`